recipe/recipe.yaml

```
@@ -22,17 +26,25 @@ requirements:
   build:
     - ${{ stdlib('c') }}
     - ${{ compiler('cxx') }}
-    - automake
-    - libtool
+    - swig
+    - cmake
     - make
-    - pkg-config
-    - gnuconfig
     - cgal
+    - if: build_platform != target_platform
+      then: python
+    - if: build_platform != target_platform
+      then: gmp
+    - if: build_platform != target_platform
+      then: mpfr
```

```suggestion
    - if: build_platform != target_platform
      then:
        - python
        - cross-python_${{ target_platform }}
        - gmp
        - mpfr
```

Can lump these together, and need `cross-python_${{ target_platform }}`, though should actually combine them with the 

```
    - if: build_platform != target_platform
      then: siscone
```

below.

---

fixed up!